docs/guides/dasharo-reviewers-guide.md

```diff
@@ -0,0 +1,223 @@
+solutions that run on the same hardware platform. The lack of performance
+testing is because, for most of the interested parties, the selling point of
+coreboot was the transparency of being open-source, and this rarely made direct
+comparisons about other features or functionality. Historically, most of what
+we could find about coreboot performance involves the lower POST times claims,
+but there needs to be more information regarding its runtime performance as
+measured in benchmarks, which is a significant point that end users are
+interested in. Sadly, we need more internal resources and the necessary field
+experience to conduct such exhaustive benchmark testing. Several significant
+findings came from third-party benchmarks and community reports, which is
+enough to know our current standing. Professional hardware reviewers should be
+better at performing and analyzing measures than we are.
+
+
+## Why We Ask To Fine Tune Settings
+
+In general, we found out in our limited benchmarking that performance
+out-of-the-box with Dasharo is measurably lower than performance with MSI
+proprietary firmware, in the order of 6-8% on the benchmarked Core i5 12600K.
+Lower performance was quickly tracked down to MSI using, by default, certain
+processor operating parameters that are quite different from the values stated
+on the Intel datasheets, resulting in alterations of the processor behavior
+that allows it to sustain Turbo Boost with higher clock speeds and for longer
+periods than otherwise possible, and thus appear to perform better overall.
+After normalizing across both MSI proprietary firmware and Dasharo, the values
+we found were different, and the benchmark difference was [reduced to around 2%
+(21:19- 29:00)](https://vimeo.com/756437712).
+
+We also found several review websites that have already written articles and
+analyses about this matter. Motherboard vendors use default values above those
+specified by Intel (technically overclocking) to produce better performance
+results in benchmarks ([AnandTech][anandtech], [TechSpot][techspot],
+[VideoCardz][videocardz], [Gamers Nexus about cheating][gamersnexus1],
+[GamersNexus about MSI default settings][gamersnexus2], [GamersNexus Z490
+YouTube Review][gamersnexus3], [TechPowerUp][techpowerup], [Tom's
+Hardware][tomshw]). Depending on whom you ask, using aggressive values by
+default can be seen either as a convenience, since the end user gets more
+performance without having to know how to configure the firmware, or as a form
+of cheating, due to the end user usually having no idea about these changes and
+that they are overclocking out-of-the-box. As for the time being, we're
+sticking to Intel specification default values; these motherboard vendors'
+custom optimizations hurts us because it makes the difference look much bigger
+than it is.
+
+Since what we want to showcase is how Dasharo (for this MSI port, coreboot +
```

Comment:
```suggestion
What we want to showcase is how Dasharo (for this MSI port, coreboot +
```

There is no subordinate clause here, no need for `since`